PantheonAddonLoader/AddonComponents/Macros.cs

```diff
@@ -28,4 +30,28 @@ public class Macros : IMacros
 
         return match is null ? null : new Macro(match);
     }
-}
\ No newline at end of file
+    public IMacro[]? GetAllMacros()
```

Comment:
```suggestion
    public IEnumerable<IMacro> GetAll()
```
`Macros.GetAll()` I think fits a bit more nicely than `Macros.GetAllMacros`

Reply:
Done